dev-tools/omdb/tests/test_all_output.rs

```diff
@@ -263,7 +305,47 @@ async fn test_omdb_env_settings(cptestctx: &ControlPlaneTestContext) {
     let args = &["--dns-server", &dns_sockaddr.to_string(), "db", "sleds"];
     do_run(&mut output, move |exec| exec, &cmd_path, args).await;
 
+    // Case: specified in multiple places (command-line argument wins)
+    let args = &["oximeter", "--oximeter-url", "junk", "list-producers"];
+    let ox = ox_url.clone();
+    do_run(
+        &mut output,
+        move |exec| exec.env("OMDB_OXIMETER_URL", &ox),
+        &cmd_path,
+        args,
+    )
+    .await;
+
+    // Case: specified in multiple places (command-line argument wins)
+    let args = &["oxql", "--metrics-db-url", "junk"];
```

Comment:
I think the CLI flag for the URL is now `--clickhouse-url`, right?

Reply:
derp. needed to push this up.